modules/qt/hm.nix

```diff
@@ -70,6 +72,9 @@
         )
         ++ (lib.optional (config.qt.style.name != recommendedStyle)
           "stylix: qt: Changing `config.qt.style` is unsupported and may result in breakage! Use with caution!"
+        )
+        ++ (lib.optional (config.qt.style.platformTheme.name == null)
+          "stylix: qt: Automatic Theme selection for Qt can only work when stylix is enabled system-wide. Set `stylix.enable = true` in `configuration.nix` .nix"
```

Comment:
IIUC, `config.qt.style.platformTheme.name` resolves to the

https://github.com/nix-community/stylix/blob/2eaa338eb879b8432f7e252d6ab8725ada98f52d/modules/qt/hm.nix#L27

type, making this state impossible.

Reply:
You're right. I should change the type here, too.